src/test/groovy/steps/DataAnalyticsSteps.groovy

```diff
@@ -181,34 +175,34 @@ class DataAnalyticsSteps {
     def xray409(count, calls){
         while (count <= calls) {
             def policyName = "new-policy"
-            xraySteps.createPolicy(policyName, username, password, xrayBaseUrl)
+            xraySteps.createPolicy(policyName, username, password, xrayURL)
```

Comment:
you really should define closures that already have anything other than _true_ variables defined

Reply:
Done